DeviceManager/TemplateHandler.py

```diff
@@ -224,28 +224,44 @@ def update_template(req, template_id):
 
         new = json_payload['attrs']
         LOGGER.debug(f" Checking old template attributes")
-        for a in old.attrs:
-            LOGGER.debug(f" Checking attribute {a}...")
-            found = False
-            for idx, b in enumerate(new):
-                LOGGER.debug(f" Comparing against new attribute {b}")
-                if (a.label == b['label']) and (a.type == b['type']):
-                    found = True
-                    a.value_type = b.get('value_type', None)
-                    a.static_value = b.get('static_value', None)
-                    new.pop(idx)
-                    LOGGER.debug(f" They match. Attribute data will be updated.")
-                    break
-            if not found:
-                LOGGER.debug(f" No match for this attribute. It will be removed.")
-                db.session.delete(a)
-
-        for a in new:
+        def attrs_match(attr_from_db, attr_from_request):
```

Comment:
I think all logs outputs from this part might also be removed, as they are too weird to be kept here. Or they can be improved to output useful data.